pkg/metrics/cluster/node_test.go

```diff
@@ -169,7 +171,7 @@ func startNode(t *testing.T, srv agentproto.ScrapingServiceServer) {
 	lcConfig.Addr = l.Addr().(*net.TCPAddr).IP.String()
 	lcConfig.Port = l.Addr().(*net.TCPAddr).Port
 
-	lc, err := ring.NewLifecycler(lcConfig, ring.NewNoopFlushTransferer(), "agent", "agent", false, prometheus.NewRegistry())
+	lc, err := ring.NewLifecycler(lcConfig, ring.NewNoopFlushTransferer(), "agent", "agent", false, logger, prometheus.NewRegistry())
```

Comment:
You can wrap here if it matters for the tests.

Reply:
Or if we don't use the registry in tests you can just pass `nil` instead of `prometheus.NewRegistry()`